spacetemplate/assets/scrum.yaml

```diff
@@ -0,0 +1,87 @@
+space_template:
+  id: "629c963a-174c-4c37-b487-272067e88bd4"
+  name: Scrum
+  description: >
+    Scrum is an agile framework for managing work with 
+    an emphasis on software development.
+work_item_types:
+- id: &impedimentID "35837274-b833-4001-bee9-4d69e7f8aa0c"
+  extends: "86af5178-9b41-469b-9096-57e5155c3f31"
+  name: Impediment
+  description: TBD
+  icon: fa fa-tasks
+- id: &issueID "22a6f261-d982-427a-a9c6-939ea398992b"
+  extends: "86af5178-9b41-469b-9096-57e5155c3f31"
+  name: Issue
+  description: TBD
+  icon: fa fa-tasks
+- id: &taskID "b8b7272e-5e1e-433d-965a-87c3e21dbed7"
+  extends: "86af5178-9b41-469b-9096-57e5155c3f31"
+  name: Task
+  description: TBD
+  icon: fa fa-tasks
+  child_types:
+  - *impedimentID
+  - *issueID
+- id: &productBacklogItemID "eb0829fc-e47e-40c5-9507-a95466e113db"
+  extends: "86af5178-9b41-469b-9096-57e5155c3f31"
+  name: Product Backlog Item
+  description: TBD
+  icon: fa fa-scissors
+  child_types:
+  - *taskID
+- id: &featureID "a661d2f7-f037-4e66-97ff-f45988791a62"
+  extends: "86af5178-9b41-469b-9096-57e5155c3f31"
+  name: Feature
+  description: TBD
+  icon: fa fa-puzzle-piece
+  child_types:
+  - *productBacklogItemID
+- id: &bugID "40335566-e207-4231-b95a-f8f5a3b941e6"
+  extends: "86af5178-9b41-469b-9096-57e5155c3f31"
+  name: Bug
+  description: TBD
+  icon: fa fa-bug
+  child_types:
+  - *productBacklogItemID
+- id: &epicID "7bd76da5-f5e3-4adf-b976-9f01c9a21416"
+  extends: "86af5178-9b41-469b-9096-57e5155c3f31"
+  name: Epic
+  description: TBD
+  icon: fa fa-university
+  child_types:
+  - *featureID
+  - *bugID
+work_item_type_groups:
+- id: "14f4dc57-2a28-4936-82a2-9d2db3735de9"        
+  name: Epics
+  type_list:
+  - *epicID
+  bucket: portfolio
+  icon: fa fa-bullseye
+- id: "1f77c6cc-19b4-4158-aa30-986a6ede47ff"        
+  name: Features
+  type_list:
+  - *featureID
+  - *bugID
+  bucket: requirement
+  icon: pficon pficon-infrastructure
+- id: "3231a8d5-15cb-4e98-a0af-8fb8c3ace915"        
+  name: BacklogItems
+  type_list:
+  - *productBacklogItemID
+  bucket: requirement
+  icon: fa fa-list-ul
+- id: "dab5e4cb-daa3-4eae-a933-b25d7fd5621b"        
+  name: Execution
+  type_list:
+  - *taskID
+  bucket: iteration
+  icon: fa fa-repeat
+work_item_link_types:
+- id: "c2c792f5-a316-4a35-a650-b17a4617625c"
+  extends: "25C326A7-6D03-4F5A-B23B-86A9EE4171E9"
+  name: "Parenting"
```

Comment:
No need to define a parenting link type. We already have one defined in the `base.yaml`. Please use that.